### ISSUE TITLE
Feature Request: Fan timeout

### ISSUE THREAD

**Issue by TheKrush** [tjjfvi/OctoPrint-BetterHeaterTimeout#5]
Have you considered adding the ability to turn off the fan as well after the heaters are shut off? My fans are always blazing away and seemed like a logical choice to build it into this plugin.

**Comment by tjjfvi** [comment]
@TheKrush Was looking through issues and came across this; I guess I missed the notification. Sorry about that!

Isn’t this already possible with the before/after gcode?

**Comment by TheKrush** [comment]
Nah, the gcode would cause the scripts to pause, I was hoping for something
that would let me start another print sometimes before the timeout, and
otherwise would operate like timing out the hotend.

- Keith (mobile)

**Comment by tjjfvi** [comment]
Are you suggesting a separate timeout for the fan, or turning the fan off after the hot end/bed timeout?
